drivers/gles2/shaders/scene.glsl

```diff
@@ -6,6 +6,7 @@
 #define mediump
 #define highp
 #else
+// Default to high precision variables for the vertex shader, no need to use medium precision here
```

Comment:
The second half of this comment is kind of confusing imo. You'd either need a longer section to explain this, or leave the second half of the comment out.

The reason why defaulting to high precision can make more sense in vertex shader, is that for vertex calculations medium precision usually isn't enough, and the trade off to use high precision makes it a better default. In the fragment shader the performance lost on mobile hardware by using high precision can be significant (if it even supports it), and many fragment shaders can get by with medium (or even low) precision, because they typically work with color values and not positions.

Reply:
```suggestion
// Default to high precision variables for the vertex shader.
// Note that the fragment shader however may default to mediump on mobile for performance,
// and thus shared uniforms should use a specifier to be consistent in both shaders.
```

Reply:
Thanks @akien!